test/test.go

```diff
@@ -32,6 +34,18 @@ func AssertImageMatches(t *testing.T, masterFilename string, img image.Image, ms
 	return test.AssertImageMatches(t, masterFilename, img, msgAndArgs...)
 }
 
+// AssertRendersToMarkup asserts that the given canvas renders the expected markup.
+// The expected markup is stripped by leading common whitespace retaining the relative indentation
+// and thus matching the indentation of the output of the used markup renderer.
+//
+// Be aware, that the indentation has to use tab characters ('\t') instead of spaces.
+// Every element starts on a new line indented one more than its parent.
+// Closing elements stand on their own line, too, using the same indentation as the opening element.
+// The only exception to this are text elements which do not contain line breaks unless the text includes them.
+func AssertRendersToMarkup(t *testing.T, expected string, c fyne.Canvas, msgAndArgs ...interface{}) bool {
```

Comment:
Can you please add
```
//
// Since: 2.0.0
```

Reply:
And If you pick this for a 1.4.x or a 1.5.0 you’ll adjust it?

Reply:
Sure thing :)